wasm.md

```diff
@@ -163,35 +215,39 @@ Note that we do not need to call `#chop` on the results here.
          </k>
 ```
 
-### Comparison Operations
+### Predicates
 
-All of the following opcodes are liftings of the K builtin operators using the helper `#bool`.
+`eqz` checks wether its operand is 0.
 
 ```k
-    syntax IUnOp ::= "eqz"
- // ----------------------
-    rule <k> ITYPE . eqz I1 => < ITYPE > #bool(I1 ==Int 0) ... </k>
+    syntax ITestOp ::= "eqz"
+ // ------------------------
+    rule <k> ITYPE . eqz I => < i32 > #bool(I ==Int 0) ... </k>
```

Comment:
Can we ignore the type here as well (ie. `_ . eqz I`) like you do below?

Reply:
Good catch, fixed.